CHANGELOG.md

```diff
@@ -1,12 +1,57 @@
 # Changelog
 
-## 7.0
+Teleport 7.0 is a major release of Teleport that contains new features, improvements, and bug fixes.
 
-Teleport 7.0 is a major release with new features, functionality, and bug fixes.
+## New Features
+
+### MongoDB 
+
+Added support for [MongoDB](https://www.mongodb.com) to Teleport Database Access. [#6600](https://github.com/gravitational/teleport/issues/6600).
+
+View the [Database Access with MongoDB](https://goteleport.com/docs/ver/7.0/database-access/guides/mongodb-self-hosted/ for more details.
+
+### AWS Console
+
+Added support for [AWS Console](https://aws.amazon.com/console) to Teleport Application Access. [#7590](https://github.com/gravitational/teleport/pull/7590)
+
+Teleport Application Access can now generate and automatically sign into the AWS Management Console using [Identity federation](https://aws.amazon.com/identity/federation).
```

Comment:
```suggestion
Teleport Application Access can now automatically sign users into the AWS Management Console using [Identity federation](https://aws.amazon.com/identity/federation). View AWS Management Console [guide](https://goteleport.com/docs/ver/7.0/application-access/guides/aws-console/) for more details.
```

Reply:
@r0mant Should we make this [guide](https://goteleport.com/docs/application-access/guides/aws-console/) as we'll push docs at the same time?